docs/scarpet/api/Auxiliary.md

```diff
@@ -142,6 +142,16 @@ Available shapes:
      * `height` - height of the cyllinder, defaults to `0`, so flat disk. Can be negative.
      * `level` - level of details, see `'sphere'`.
 
+ * `'poly'`:
+   * Required attributes:
+     * `points` - vertices of the polygon
+   * Optional attributes:
+     * `relative` - list of bools. vertices of the polygont that affected by 'follow'. Default means that every point is affacted.
+     * `mode` - how those points are connected. may be "polygon"(default),"strip" or "triangles". "polygon" means that it will be viewed as vertices of a polygon center on the first one. "strip" means that it will be viewed as a triangles strip. "triangles" means that it will be viewed as some triangles that are not related to each other.
```

Comment:
You should specify that for `triangles` mode, you need to input a triplet of vertices for each triangle